bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,257 @@
+			<channel id="temperatureOverheatingTarget2" typeId="temperatureOverheatingTarget2"/>
+			<channel id="highPressure2" typeId="highPressure2"/>
+			<channel id="lowPressure2" typeId="lowPressure2"/>
+			<channel id="inputSwitchHighPressure2" typeId="inputSwitchHighPressure2"/>
+			<channel id="outputDefrostValve2" typeId="outputDefrostValve2"/>
+			<channel id="outputVBO2" typeId="outputVBO2"/>
+			<channel id="outputCompressor1_2" typeId="outputCompressor1_2"/>
+			<channel id="outputCompressorHeating2" typeId="outputCompressorHeating2"/>
+			<channel id="secondShutdownReason0" typeId="secondShutdownReason0"/>
+			<channel id="secondShutdownReason1" typeId="secondShutdownReason1"/>
+			<channel id="secondShutdownReason2" typeId="secondShutdownReason2"/>
+			<channel id="secondShutdownReason3" typeId="secondShutdownReason3"/>
+			<channel id="secondShutdownReason4" typeId="secondShutdownReason4"/>
+			<channel id="secondShutdownTime0" typeId="secondShutdownTime0"/>
+			<channel id="secondShutdownTime1" typeId="secondShutdownTime1"/>
+			<channel id="secondShutdownTime2" typeId="secondShutdownTime2"/>
+			<channel id="secondShutdownTime3" typeId="secondShutdownTime3"/>
+			<channel id="secondShutdownTime4" typeId="secondShutdownTime4"/>
+			<channel id="temperatureRoom" typeId="temperatureRoom"/>
+			<channel id="temperatureRoomTarget" typeId="temperatureRoomTarget"/>
+			<channel id="temperatureHotWaterTop" typeId="temperatureHotWaterTop"/>
+			<channel id="frequencyCompressor" typeId="frequencyCompressor"/>
+
+			<channel id="temperatureHeatingParallelShift" typeId="temperatureHeatingParallelShift"/>
+			<channel id="heatingMode" typeId="heatingMode"/>
+			<channel id="hotWaterMode" typeId="hotWaterMode"/>
+			<channel id="comfortCoolingMode" typeId="comfortCoolingMode"/>
+			<channel id="thermalDisinfectionMonday" typeId="thermalDisinfectionMonday"/>
+			<channel id="thermalDisinfectionTuesday" typeId="thermalDisinfectionTuesday"/>
+			<channel id="thermalDisinfectionWednesday" typeId="thermalDisinfectionWednesday"/>
+			<channel id="thermalDisinfectionThursday" typeId="thermalDisinfectionThursday"/>
+			<channel id="thermalDisinfectionFriday" typeId="thermalDisinfectionFriday"/>
+			<channel id="thermalDisinfectionSaturday" typeId="thermalDisinfectionSaturday"/>
+			<channel id="thermalDisinfectionSunday" typeId="thermalDisinfectionSunday"/>
+			<channel id="thermalDisinfectionPermanent" typeId="thermalDisinfectionPermanent"/>
+			<channel id="temperatureComfortCoolingATRelease" typeId="temperatureComfortCoolingATRelease"/>
+			<channel id="temperatureComfortCoolingATReleaseTarget" typeId="temperatureComfortCoolingATReleaseTarget"/>
+			<channel id="comfortCoolingATExcess" typeId="comfortCoolingATExcess"/>
+			<channel id="comfortCoolingATUndercut" typeId="comfortCoolingATUndercut"/>
+		</channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>IP address</label>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.

Reply:
ah cool. will try that.